openslides/agenda/static/js/agenda/site.js

```diff
@@ -431,6 +431,9 @@ angular.module('OpenSlidesApp.agenda.site', [
         // Return true if the requested user is allowed to do a specific action
         // and see the corresponding button (e.g. 'add me' or 'remove me').
         $scope.isAllowed = function (action) {
+            if (!operator.user) {
```

Comment:
Why should anonymous not be allowed to start or stop speakers (if the default group has this permission of course)?